units/src/parse.rs

```diff
@@ -174,6 +175,7 @@ macro_rules! impl_tryfrom_str_from_int_infallible {
 ///
 /// All functions use the error returned by `$inner_fn`.
 #[macro_export]
+#[doc(hidden)] // This is an 'internal' macro that should not be used outside of the `rust-bitcoin` crate.
```

Comment:
No, this needs to be moved to `internals` instead. If anything else than **a public** macro from **this** crate uses it then `hidden` doesn't work.

Reply:
Yeah, derp, oops.

Reply:
What do you mean by 'doesn't work'? I thought the intention of `doc(hidden)` was "this is public but we don't guarantee it so we pretend its not public by hiding it from the docs". And from recent discussion a month or so ago that is standard Rust practice. That was my understanding from the conversation anyway.

Reply:
We need to centralize this discussion somewhere.

Reply:
I just did that yesterday so I totally do not understand you argument man?

Reply:
> We need to centralize this discussion somewhere.

Woops, I'll stop posting here now.